src/styles/issues.module.scss

```diff
@@ -1,6 +1,9 @@
 $link-text: #2563eb;
 $link-bg: #dbeafe;
 $border-color: #d9d9d9;
+$color-blue: #041187;
```

Comment:
can we have better name for this variable, so we know what it is meant for

Reply:
This is a generic color type and does not contain any context.